docker-compose-migration.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.7"
+services:
+  db:
+      image: postgres:13
+      env_file:
+         - ./.env
+      healthcheck:
+        test: ["CMD", "pg_isready", "-U", "mobilizon_reshare"]
+        interval: 5s
+        retries: 5
+      ports:
+        - 5432:5432
```

Comment:
Can't we use `docker-compose-web.yml` with `docker-compose up -d db` instead of having two different files?

Reply:
I want the other one to have a volume by default and this not to have it. We could drop volumes in the migration script but that sounds even worse.  Also at some point they might diverge even more.